frontend/src/assets/manual/en-GB/query.md

```diff
@@ -47,6 +59,7 @@ Illustrating the differences when searching for different combinations of `bank`
 | `bank~1` | 76241 hits (compare with just bank) |
 | `"the bank is"` | 24 hits |
 | `"the bank is" ~1`| 32 hits |
+| `"interest balance"~5 "balance interest"~5` | 285 hits |
```

Comment:
this is almost identical to the row above it, but switches to a different example. Suggestion for clarity:

```suggestion
| `"the bank is"~5 "is the bank"~5` | 285 hits |
```

Reply:
I decided to rewrite this and the above examples to _asset_ / _bank_ combinations, as that is what the text on top of the section states.